### PR TITLE
add cassandra tool support for user and password connection to cluster

### PR DESCRIPTION
**Pull request by samarabbas**

### CODE REVIEW

File: tools/cassandra/setupTask.go [hunk 1]
```diff
@@ -33,7 +33,8 @@ type SetupSchemaTask struct {
 }
 
 func newSetupSchemaTask(config *SetupSchemaConfig) (*SetupSchemaTask, error) {
-	client, err := newCQLClient(config.CassHosts, config.CassKeyspace)
+	client, err := newCQLClient(config.CassHosts, config.CassPort, config.CassUser, config.CassPassword,
+		config.CassKeyspace)
```
Comment (venkat1109) [inline tools/cassandra/setupTask.go]: remove newline

File: tools/cassandra/handler.go [hunk 2]
```diff
@@ -82,6 +82,9 @@ func handleUpdateSchema(config *UpdateSchemaConfig) error {
 }
 
 func handleSetupSchema(config *SetupSchemaConfig) error {
+	if config.CassPort == 0 {
```
Comment (venkat1109) [inline tools/cassandra/handler.go]: move this inside validateSetupSchemaConfig() - also add the same check to validateUpdateSchemaConfig and validateCreateKeyspaceConfig for consistency

File: tools/cassandra/setupTask.go (hunk 1 above)
Comment (samarabbas) [inline tools/cassandra/setupTask.go]: I usually wrap around lines at 120 characters.

File: tools/cassandra/handler.go (hunk 2 above)
Comment (samarabbas) [inline tools/cassandra/handler.go]: fixed.

File: tools/cassandra/setupTask.go (hunk 1 above)
Comment (venkat1109) [inline tools/cassandra/setupTask.go]: yeah, 120 chars is the correct config, there was an additional newline.. May be its just the diff viewer, ignore.

**Pull request merged**